packages/office-ui-fabric-react/src/components/List/List.tsx

```diff
@@ -93,7 +95,8 @@ export class List<T = any> extends BaseComponent<IListProps<T>, IListState<T>> i
 
   private _root = React.createRef<HTMLDivElement>();
   private _surface = React.createRef<HTMLDivElement>();
-
+  private _events: EventGroup;
```

Comment:
We can use `on` here. I'll push an iteration.

Reply:
Resolved in https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/c785c906e7ccdd750163c95b7d785fdb9cec4a9e.

Reply:
Looks like the above caused unit test failures (thanks tests). I'll take a look to see if the issue is easily resolvable to unblock PR.